spec/requests/api/curriculums_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'swagger_helper'
+
+describe 'Curriculums API' do
+
+  path '/api/curriculums' do
+    get 'Returns all Curriculums' do
+      tags 'Curriculums'
+      produces 'application/json'
+
+      response '200', 'Success' do
+        schema type: :array,
+        items: {
+          type: :object,
+          properties: {
+            id: {type: :integer},
+            name: {type: :string}
+          }
+        }
+        run_test!
+      end
+    end
+  end
+
+  path '/api/curriculums/{id}' do
+    get 'Returns Curriculum' do
+      tags 'Curriculums'
+      parameter name: :id, in: :path, type: :string
+      produces 'application/json'
+
+      response '200', 'Success' do
+        schema type: :object,
+        properties: {
+          id: {type: :integer},
+          name: {type: :string}
+        }
+        run_test!
+      end
+    end
+  end
+
+  path '/api/curriculums/{curriculum_id}/learning_units' do
+    get 'Returns all Learning Unit from a Curriculum' do
+      tags 'Learning Units'
+      produces 'application/json'
+      parameter name: :curriculum_id, in: :path, type: :string
+
+      response '200', 'Success' do
+        schema type: :array,
+        items: {
+          type: :object,
+          properties: {
+            id: {type: :integer},
+            name: {type: :string}
+          }
+        }
+        run_test!
+      end
+    end
+  end
+
+  path '/api/learning_units/{id}' do
```

Comment:
Should be in a different spec called learning_units_spec

Reply:
Done